progenitor-impl/src/lib.rs

```diff
@@ -361,6 +366,8 @@ impl Generator {
                     RequestBuilderExt,
                     ResponseValue,
                 };
+                #[allow(unused_imports)]
+                use std::convert::TryInto;
```

Comment:
Is this needed? All the uses of `.try_into()` we generate are on generic types that explicitly have a `TryInto` trait bound and thus don't require importing the trait.

Reply:
evidently not! it was in a previous incarnation and i neglected to try removing it again, thanks